docs/extra-packages-ref.rst

```diff
@@ -26,13 +26,17 @@ Here's the list of the :ref:`subpackages <installation:extra_packages>` and what
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
 | subpackage          | install command                                     | enables                                                              |
 +=====================+=====================================================+======================================================================+
-| all                 | ``pip install 'apache-airflow[all]'``               | All Airflow features known to man                                    |
+| all                 | ``pip install 'apache-airflow[all]'``               | All Airflow user facing features (no devel and doc requirements)     |
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
 | all_dbs             | ``pip install 'apache-airflow[all_dbs]'``           | All databases integrations                                           |
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
-| devel               | ``pip install 'apache-airflow[devel]'``             | Minimum dev tools requirements                                       |
+| devel               | ``pip install 'apache-airflow[devel]'``             | Minimum dev tools requirements (includes docs tools and ``all``)     |
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
-| devel_hadoop        | ``pip install 'apache-airflow[devel_hadoop]'``      | Airflow + dependencies on the Hadoop stack                           |
+| devel_hadoop        | ``pip install 'apache-airflow[devel_hadoop]'``      | Same as ``devel`` + dependencies for developing the Hadoop stack     |
++---------------------+-----------------------------------------------------+----------------------------------------------------------------------+
+| devel_all           | ``pip install 'apache-airflow[devel_all]'``         | Same as ``devel`` + additional requirements like ``snakebite``       |
```

Comment:
I'm not familiar with `snakebite` but:
> Snakebite is a python library that provides a pure python HDFS client

Suggest that we may consider putting it into `devel_hadoop` as there are already `hdfs` packages?